src/common/lib/pinata-proxy.js

```diff
@@ -17,31 +15,15 @@ export const uploadFile = val => {
     pinataOptions: { cidVersion: 0 }
   }
 
-  const CancelToken = axios.CancelToken
-  const source = CancelToken.source()
-
-  store.dispatch("geneticData/setCancel", {
-    source
-  })
-
   return pinataIpfsUploadFile(
     options,
     val.file,
-    pinataJwtKey,
-    source.token,
-    (progressEvent) => {
-      if(progressEvent.lengthComputable) {
-        let percentCompleted = Math.round( (progressEvent.loaded * 100) / progressEvent.total )
-        store.dispatch("geneticData/getLoadingProgress", {
-          progress: percentCompleted
-        })
-      }
-    }
```

Comment:
just curious, why did you delete these codes? and have you checked the whole environment does not work properly if these codes are removed?

Reply:
because previously there was an error in cancelToken function, and after I checked this function was just used on this page

Reply:
what kind of error? can you mention the error here? because I doubt it will run properly on other pages

Reply:
oh Sorry, after I checked again, it turned out that there was no error. i will revert the code

Reply:
updated @bukandicki